TCL4 Data Management/utm-tcl4-dmp-cns.yaml

```diff
@@ -36,85 +35,58 @@ info:
+        type: array
+        items: 
+          $ref: "#/definitions/UasTruthXYZEcefCoordinates_ft"
       emergencyPhase_nonDim:
         description: "During operation, operator must monitor UA operation according\
           \ to Communications and Navigation Procedure (Reference 9, Section 4.2)\
           \ and report appropriate emergency phase with UTC timestamp as enum"
-        $ref: "#/definitions/TimestampedEmergencyPhase_nonDim_Enum"
+        type: array
+        items: 
+          $ref: "#/definitions/TimestampedEmergencyPhase_nonDim_Enum"
       offNominalMitigationOutcome_nonDim:
         description: "Outcome of offnominal events like UA to Operator C2 Link Loss,\
           \ Operator to UA C2 Link Loss, or Loss of Vehicle Navigation, is reported\
           \ as enum"
-        $ref: "#/definitions/TimestampedOffNominalMitigationOutcome_nonDim_Enum"
+        type: array
+        items: 
+          $ref: "#/definitions/TimestampedOffNominalMitigationOutcome_nonDim_Enum"
       landingOutcome_nonDim:
         description: "Report this for every landing as enum, landing within contingency_polygon\
           \ is also considered as safe landing"
-        $ref: "#/definitions/TimestampedLandingOutcome_nonDim_Enum"
+        type: array
+        items: 
+          $ref: "#/definitions/TimestampedLandingOutcome_nonDim_Enum"
+      minimumAircraftToOperatorDataTransferRateForOperation_bytesPerSec:
+        description: "Report the minimum aircraft to operator data transfer rate (in Bytes per second) necessary to execute the operation. Consider that data should be authenticated and checked for integrity. (bytesPerSec)"
+        type: integer
 
+      minimumOperatorToAircraftDataTransferRateForOperation_bytesPerSec:
+        description: "Report the minimum aircraft to operator data transfer rate (in Bytes per second) necessary to execute the operation. Consider that data should be authenticated and checked for integrity. (bytesPerSec)"
+        type: integer
 
-      aircraftToOperatorDataTransferRate_bytesPerSec:
-        description: "Report aircraft to operator data transfer rate (in Bytes per\
-        \ second), data must be authenticated and checked for integrity. (bytesPerSec)"
-        $ref: "#/definitions/TimestampedInteger"
-          
-      aircraftToOperatorRoundTripLatency_ms:
-        description: "Report Round-trip latency from source to destination, in millisecond\
-        \ (ms)"
-        $ref: "#/definitions/TimestampedInteger"
-      operatorToAircraftRoundTripLatency_ms:
-        description: "Report Round-trip latency from source to destination, in millisecond\
-          \ (ms)"
-        $ref: "#/definitions/TimestampedInteger"
+      maximumAircraftToOperatorRoundTripLatencyTolerableForOperation_ms:
+        description: "Report the maximum tolerable Round-trip latency from source to destination to execute the operation, in millisecond (ms)"
+        type: number
```

Comment:
we are asking for ms.  Are you expecting more precision than that?  If not, this should be an integer type. 

While you are here, add `minimum: 0`